test/test_source_epss.py

```diff
@@ -12,15 +13,26 @@ def setup_class(cls):
         )
 
     final_data = [
-        ("CVE-1999-0001", "EPSS", "0.011", "0.82987"),
-        ("CVE-2019-10354", "EPSS", "0.00287", "0.64385"),
-        ("CVE-1999-0003", "EPSS", "0.999", "0.88555"),
-        ("CVE-2023-28143", "EPSS", "0.00042", "0.05685"),
-        ("CVE-2017-15360", "EPSS", "0.00078", "0.31839"),
-        ("CVE-2008-4444", "EPSS", "0.07687", "0.93225"),
-        ("CVE-1999-0007", "EPSS", "0.00180", "0.54020"),
+        ("CVE-1999-0001", 1, "0.011", "0.82987"),
+        ("CVE-2019-10354", 1, "0.00287", "0.64385"),
+        ("CVE-1999-0003", 1, "0.999", "0.88555"),
+        ("CVE-2023-28143", 1, "0.00042", "0.05685"),
+        ("CVE-2017-15360", 1, "0.00078", "0.31839"),
+        ("CVE-2008-4444", 1, "0.07687", "0.93225"),
+        ("CVE-1999-0007", 1, "0.00180", "0.54020"),
     ]
 
     def test_parse_epss(self):
+        # EPSS need metrics table to populated in the database. To get the EPSS metric id from table.
```

Comment:
We can convert this comment to a docstring too.

```suggestion
        ''' EPSS need metrics table to populated in the database. To get the EPSS metric id from table. '''
```

Honestly, I feel like using the docstring format matters more in the code than the tests because it'll allow for hinting in dev environments and auto fills out info in API views.  But it still feels like there's no reason not to just be in the habit of using docstrings always if we can. 

You might need to change the line wraps on this particular one for black/flake8 compliance rather than just hitting accept on the suggestion.  I'm too lazy to count characters manually but the tools will tell you!